api/services/control/control.proto

```diff
@@ -53,17 +53,23 @@ message UsageRecord {
 	bool Shared = 11;
 }
 
+message ExporterAttrs {
+	map<string, string> exporterAttrs = 1;
+}
+
 message SolveRequest {
 	string Ref = 1;
 	pb.Definition Definition = 2;
 	string Exporter = 3;
-	map<string, string> ExporterAttrs = 4;
-	string Session = 5;
-	string Frontend = 6;
-	map<string, string> FrontendAttrs = 7;
-	CacheOptions Cache = 8 [(gogoproto.nullable) = false];
-	repeated string Entitlements = 9 [(gogoproto.customtype) = "github.com/moby/buildkit/util/entitlements.Entitlement" ];
-	map<string, pb.Definition> FrontendInputs = 10;
```

Comment:
We shouldn't break API compatibility

Reply:
@AkihiroSuda are you talking about introducing new type `ExporterAttrs` ? If so, I did it because we can't do  ` repeated map<string, string>` .

Reply:
Not only ExportAttrs. All fields need to keep same field number and type.

Reply:
@AkihiroSuda done.